homeassistant/components/tado/tado_connector.py

```diff
@@ -221,6 +240,8 @@ def update_home(self):
 
     def get_capabilities(self, zone_id):
         """Return the capabilities of the devices."""
+        if self.is_x:
+            return {"type": TYPE_HEATING}
```

Comment:
Why does the library not provide this information for the TadoX too?

Reply:
That is because the Tado X API does not return any comparable capabilities. The endpoint that did this in the old API is not existing anymore.

Reply:
Same issue as above. `get_capabilities` is an Endpoint of the Tado pre-X devices. 

The library should abstract this implementation detail away, so the HA integration wouldn't have to know about this.

Reply:
The issue is that there is a [complete rewrite](https://github.com/erwindouna/python-tado) being done by @erwindouna at the time anyway, so from our point of view it isn't really a good use of time to make everything "beautiful" in the old library. This here is intended as a rather quick fix so users can get their Tado X working with the beginning of the heating season in the northern hemisphere.